Sources/Filters/General/MoleculeToRepresentation/index.js

```diff
@@ -190,6 +300,7 @@ export function extend(publicAPI, model, initialValues = {}) {
   macro.setGet(publicAPI, model, [
     'sphereScaleArrayName',
     'radiusType',
+    'hideElement',
```

Comment:
You should use

https://github.com/Kitware/vtk-js/blob/master/Sources/macro.js#L283-L316

But you may have to fix/improve it so you could provide a size of -1 in case you don't know it so it does not throw an exception.